scrapy_zyte_api/__version__.py

```diff
@@ -0,0 +1 @@
+__version__ = "0.11.1"
```

Comment:
:sweat_smile: 

```suggestion
__version__ = "0.12.0"
```

Reply:
Oh, haven't noticed that it was updated, thanks @Gallaecio !